examples/framesjs-starter/app/examples/simplified-api/[[...routes]]/route.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnyJson } from "../new-api";
+import {
+  FrameImage,
+  Frame,
+  FramesApp,
+  FrameLink,
+  renderFramesToResponse,
+} from "../react-api";
+
+type FrameState = {
+  clicked: number;
+};
+
+function isFrameState(
+  frameState: AnyJson | undefined
+): frameState is FrameState {
+  if (frameState && typeof frameState === "object" && "clcked" in frameState) {
+    return typeof frameState.clicked === "number";
+  }
+
+  return false;
+}
+
+export const framesApp = (
+  <FramesApp>
+    <Frame index>
```

Comment:
Where do you execute side effects like updating a db or executing a transaction?

Reply:
That one is tricky and currently it is not implemented and I think that's the drawback of using React API as it doesn't allow us to use async render handlers. 

Either we would have some sort of "hacky" solution I mean you could have on `Frame` a prop `handler: () => Promise<void>` or something like that.

Otherwise on agnostic API it would be just a handler property where you could define your own async function to do something when Frame is rendering.